core/server/api/settings.js

```diff
@@ -434,19 +386,27 @@ settings = {
 module.exports = settings;
 
 /**
- * synchronous function to get cached settings value
- * returns the value of the settings entry
+ * @TODO:
+ * - move settings cache somewhere else and listen on model changes
+ * - why are the settingsCache keys without type (core|blog)
```

Comment:
the theme config cache thing that was here before only ever included the settings of type blog so that information was automatically assumed.